dotCMS/src/integration-test/java/com/dotmarketing/quartz/DotStatefulJobTest.java

```diff
@@ -110,7 +110,7 @@ public void Test_Launch_Stateful_Jobs_Verify_They_Dont_Overlap_In_Time()
     private Optional<JobExecutionContext> getJobExecutionContext(){
         final String jobName = getJobName(MyStatefulJob.class);
         try {
-            final Scheduler sequentialScheduler = getSequentialScheduler();
+            final Scheduler sequentialScheduler = DotSchedulerFactory.getInstance().getScheduler();
```

Comment:
Theoretically this class `CronScheduledTask` has a param that should instruct before what scheduler needed to be used. Sequential/Standart maybe that para  and the getter should be removed or at least marked deprecated